cmd/check_container.go

```diff
@@ -152,111 +152,117 @@ var checkContainerCmd = &cobra.Command{
 			return err
 		}
 
-		// submitting results to pxysis if submit flag is set
+		// assemble artifacts and submit results to pyxis if user provided the submit flag.
 		if submit {
 			log.Info("preparing results that will be submitted to Red Hat")
-			log.Tracef("CertProject: %+v", certProject)
+			log.Tracef("CertProject: %+v", certProject) // TODO() This depends on Pyxis calls made outside of this block.
```

Comment:
My proposed changes to the UX would move those calls to here.

Reply:
💯 Indeed. When that merges, these notes should go away.